src/NextGen/identity/identity.ts

```diff
@@ -0,0 +1,59 @@
+import { MPID, User } from '@mparticle/web-sdk';
+import { IMParticleComponent } from '../core/component';
+import { IdentityCallback } from './identityCallback';
+
+// QUESTION: What is the diff between this and IdentityApiData?
```

Comment:
I'd say IdentityApiRequest is roughly equivalent to what I assume IdentityApiData is? On mobile a request class usually contains the post data like what Identities are being changed https://github.com/mParticle/mparticle-apple-sdk/blob/main/mParticle-Apple-SDK/Include/MPIdentityApiRequest.h